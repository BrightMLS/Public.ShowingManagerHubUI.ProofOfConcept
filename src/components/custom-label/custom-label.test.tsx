import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import CustomLabel from './custom-label'

describe('<CustomLabel /> Component', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(
      <CustomLabel name="Application ID" isRequired={true} />,
    )
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})

describe('<CustomLabel /> Component', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(
      <CustomLabel name="Application ID" isRequired={false} />,
    )
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
