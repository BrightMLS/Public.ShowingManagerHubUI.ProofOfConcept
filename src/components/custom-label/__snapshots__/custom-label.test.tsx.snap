// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CustomLabel /> Component should render without crashing 1`] = `
<WithStyles(ForwardRef(Typography))
  className="makeStyles-placeHolder-1"
  variant="inherit"
>
  Application ID*
</WithStyles(ForwardRef(Typography))>
`;

exports[`<CustomLabel /> Component should render without crashing 2`] = `
<WithStyles(ForwardRef(Typography))
  className="makeStyles-placeHolder-1"
  variant="inherit"
>
  Application ID
</WithStyles(ForwardRef(Typography))>
`;
